src/components/backend-ai-console.ts

```diff
@@ -1116,25 +1260,43 @@ export default class BackendAIConsole extends connect(store)(LitElement) {
+          <mwc-textfield id="pref-original-name" type="text"
+              label="${_t('console.menu.FullName')}" max-length="30" autofocus
+              style="margin-bottom:20px;" value="${this.full_name}">
+          </mwc-text-field>
+        </div>
         <div slot="content" class="layout vertical" style="width:300px;">
           <mwc-textfield id="pref-original-password" type="password"
-              label="${_t('console.menu.OriginalPassword')}" max-length="30" autofocus
-              style="margin-bottom:20px">
-          </mwc-textfield>
-          <mwc-textfield id="pref-new-password" label="${_t('console.menu.NewPassword')}"
-              type="password" min-length="8" max-length="30"
-              auto-validate validationMessage="${_t('console.menu.InvalidPasswordMessage')}"
-              pattern="^(?=.*[A-Za-z])(?=.*\\d)(?=.*[@$!%*#?&])[A-Za-z\\d@$!%*#?&]{8,}$">
-          </mwc-textfield>
-          <mwc-textfield id="pref-new-password2" label="${_t('console.menu.NewPasswordAgain')}"
-              type="password" min-length="8" max-length="30">
+              label="${_t('console.menu.OriginalPassword')}" max-length="30"
+              style="margin-bottom:20px;">
           </mwc-textfield>
+          <div class="horizontal flex layout">
+            <mwc-textfield id="pref-new-password" label="${_t('console.menu.NewPassword')}"
+                type="password" min-length="8" max-length="30"
+                auto-validate validationMessage="${_t('console.menu.InvalidPasswordMessage')}"
+                pattern="^(?=.*[A-Za-z])(?=.*\\d)(?=.*[@$!%*#?&])[A-Za-z\\d@$!%*#?&]{8,}$"
+                @change="${this._validatePassword}">
+            </mwc-textfield>
+            <mwc-icon-button-toggle off onIcon="visibility" offIcon="visibility_off"
+                                      @click="${(e) => this._togglePasswordVisibility(e.target)}">
+            </mwc-icon-button-toggle>   
+          </div>
+          <div class="horizontal flex layout">
+            <mwc-textfield id="pref-new-password2" label="${_t('console.menu.NewPasswordAgain')}"
+                type="password" min-length="8" max-length="30"
+                @change="${this._validatePassword}">
+            </mwc-textfield>
+            <mwc-icon-button-toggle off onIcon="visibility" offIcon="visibility_off"
+                                      @click="${(e) => this._togglePasswordVisibility(e.target)}">
+              </mwc-icon-button-toggle>   
+          </div>
         </div>
         <div slot="footer" class="horizontal end-justified flex layout">
           <div class="flex"></div>
           <wl-button class="cancel" inverted flat @click="${this._hideUserPrefDialog}">${_t("console.menu.Cancel")}</wl-button>
-          <wl-button class="ok" @click="${this._updateUserPassword}">${_t("console.menu.Update")}</wl-button>
+          <wl-button class="ok" @click="${this._updateUserInformation}">${_t("console.menu.Update")}</wl-button>
```

Comment:
It should be changed to `$(()->this._updateUserInformation()}` while it is automatically run when page is loaded in some circumstances.